templates/common/baremetal/files/baremetal-keepalived.yaml

```diff
@@ -57,29 +34,79 @@ contents:
           - name: NSS_SDB_USE_CACHE
             value: "no"
         command:
-        - /usr/sbin/keepalived
-        args:
-        - "-f"
-        - "/etc/keepalived/keepalived.conf"
-        - "--dont-fork"
-        - "--vrrp"
-        - "--log-detail"
-        - "--log-console"
+        - /bin/bash
+        - -c
+        - |
+          #/bin/bash
+          reload_keepalived()
+          {
+            declare -r pid="$(pgrep -o keepalived)"
+            kill -s SIGHUP "$pid"
+          }
+
+          msg_handler()
+          {
+            while read -r line; do
+              echo "The client sent: $line"  >&2
```

Comment:
there is a possibility so safe one (!) character by dropping the extra space

Reply:
Done